http_service/pkg/http_user_service/repository.go

```diff
@@ -0,0 +1,38 @@
+package httpuserservice
+
+import (
+	"context"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+	"github.com/jcromanu/final_project/http_service/errors"
+	"github.com/jcromanu/final_project/http_service/pkg/entities"
+	"github.com/jcromanu/final_project/user_service/pb"
+	"google.golang.org/grpc"
+)
+
+type HttpRepository interface {
+	CreateUser(context.Context, entities.User) (int32, error)
+}
+
+type httpRespository struct {
```

Comment:
the repo is a grpc repository not an convetional http, can you rename as proxy or grpcrepo